### PR TITLE
Bug 1771522: add version and skiprange to operatorbundle table

### PR DESCRIPTION
**Pull request by gallettilance**
**Description of the change:**

**Motivation for the change:**

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

**Comment by dinhxuanvu** [comment]
lgtm

File: pkg/sqlite/load.go [hunk 1]
```diff
@@ -639,7 +655,21 @@ func (s *SQLLoader) AddBundlePackageChannels(manifest registry.PackageManifest,
...
-	if _, err := stmt.Exec(csvName, csvBytes, bundleBytes, bundleImage); err != nil {
+	csv, err := bundle.ClusterServiceVersion()
+	if err != nil {
+		return err
+	}
+
+	skiprange, ok := csv.Annotations[SkipRangeAnnotationKey]
+	if !ok {
+		skiprange = ""
+	}
```
Comment (njhale) [inline pkg/sqlite/load.go]: What do you think about having a `SkipRange()` method on the `ClusterServiceVersion` struct and handling this there?

File: pkg/sqlite/load.go [hunk 2]
```diff
@@ -77,7 +79,21 @@ func (s *SQLLoader) AddOperatorBundle(bundle *registry.Bundle) error {
...
+	version, err := csv.GetVersion()
+	if err != nil {
+		version = ""
+	}
```
Comment (njhale) [inline pkg/sqlite/load.go]: It looks like we're doing this in multiple places. Can we surface these as methods of `registry.Bundle()` and memoize this work?

**Pull request merged**